en/docs/install-and-setup/install/installation-prerequisites.md

```diff
@@ -87,6 +87,53 @@ If you have difficulty in setting up any WSO2 product in a specific platform or
 </ul>
 </td></tr></table></html>
 
+## Database Storage Requirements
```

Comment:
Sub-heading are written in sentence case and only the page heading is written in Title case. So in this case "Storage Requirements" should start with lower case.